packager/packager.cc

```diff
@@ -631,9 +890,9 @@ vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers
                 if (result.gotItem()) {
                     filesProcessed++;
                     if (job.file.data(gs).sourceType == core::File::Type::Normal) {
-                        core::Context ctx(gs, core::Symbols::root(), job.file);
+                        core::MutableContext ctx(gs, core::Symbols::root(), job.file);
```

Comment:
What changed that a `MutableContext` is required now?

Reply:
Ahh, I think I thought you needed Mutable for errors in a tree walk. Fixing.